qdrant_client/qdrant_fastembed.py

```diff
@@ -0,0 +1,287 @@
+                And then used as a query vector.
+            query_filter:
+                - Exclude vectors which doesn't fit given conditions.
+                - If `None` - search among all vectors
+            search_params: Additional search params
+            limit: How many results return
+            offset:
+                Offset of the first result to return.
+                May be used to paginate results.
+                Note: large offset values may cause performance issues.
+            with_payload:
+                - Specify which stored payload should be attached to the result.
+                - If `True` - attach all payload
+                - If `False` - do not attach any payload
+                - If List of string - include only specified fields
+                - If `PayloadSelector` - use explicit rules
+            with_vectors:
+                - If `True` - Attach stored vector to the search result.
+                - If `False` - Do not attach vector.
+                - If List of string - include only specified fields
+                - Default: `False`
+            score_threshold:
+                Define a minimal score threshold for the result.
+                If defined, less similar results will not be returned.
+                Score of the returned result might be higher or smaller than the threshold depending
+                on the Distance function used.
+                E.g. for cosine similarity only higher scores will be returned.
+            embedding_model:
+                Which embedding model to use. Defaults to "fast-all-MiniLM-L6-v2".
+                Models are automatically downloaded by `fastembed` library.
+
+        Returns:
+            List[types.ScoredPoint]: List of scored points.
+
+        """
+        embedding_model_inst = self._get_or_init_model(model_name=embedding_model)
+        embeddings = list(embedding_model_inst.encode(documents=[query_text]))
+        query_vector = embeddings[0][0]
+
+        return self.search(
+            collection_name=collection_name,
+            query_vector=query_vector,
+            query_filter=query_filter,
+            search_params=search_params,
+            limit=limit,
+            offset=offset,
+            with_payload=with_payload,
+            with_vectors=with_vectors,
+            score_threshold=score_threshold,
+            **kwargs,
+        )
+
+    def query_batch(
+        self,
+        collection_name: str,
+        query_texts: List[str],
+        query_filter: Optional[models.Filter] = None,
+        search_params: Optional[models.SearchParams] = None,
+        limit: int = 10,
+        offset: int = 0,
+        with_payload: Union[bool, Sequence[str], models.PayloadSelector] = True,
+        with_vectors: Union[bool, Sequence[str]] = True,
+        score_threshold: Optional[float] = None,
+        embedding_model: str = DEFAULT_EMBEDDING_MODEL,
+        **kwargs: Any,
+    ) -> List[List[types.ScoredPoint]]:
```

Comment:
The most notable difference is that this accepts a `List[str]` as queries — this is not worth learning for most MLE! 

Can we make a call from the `query` endpoint to support this behaviour instead? Thereby removing the need for engineer to learn this — and keeping it as a separate function at the same time?